plugin/db/sqlite/sqlite.go

```diff
@@ -37,13 +51,22 @@ func newDriver(config db.DriverConfig) db.Driver {
 
 // Open opens a SQLite driver.
 func (driver *Driver) Open(ctx context.Context, dbType db.Type, config db.ConnectionConfig, connCtx db.ConnectionContext) (db.Driver, error) {
-	// TODO(spinningbot): implement it.
-	return nil, nil
+	// Host is the directory (instance) containing all SQLite databases.
+	driver.dir = config.Host
+	// We assume each instance to have a default.db database.
```

Comment:
Not quite get this logic. How we make sure each dir contains a default.db?

If we want to get a connection, we can also use its in memory db or temporary db https://www.sqlite.org/inmemorydb.html

Reply:
driver.GetDbConnection(ctx, "default") will create "default.db" file. The default database option kinda follows Postgres behavior of having a default database called "postgres". But I'm okay with one way or the other.

Reply:
I guess using memdb maybe the best way to reduce side-effect while getting a valid handle

Reply:
SG. Done.